src/mixins/i18n-sync.js

```diff
@@ -0,0 +1,38 @@
+import { sendWindowMessage } from '~/utils/send-message.js'
+import config from '../../nuxt.config.js'
+
+export default {
+  data() {
+    return {
+      /**
+       * Handles messages of type `localeSet` received by the `iframe`. Any
+       * other message types will by discarded.
+       *
+       * @param {string} type - the nature of the message received
+       * @param {{ locale: string, lang: string }} value - the data sent with the message
+       */
+      localeMsgHandler: ({ data: { type, value } }) => {
```

Comment:
Just for my own understanding, why is this function defined in `data` instead of in `methods`?

Reply:
You're absolutely right about this. It should be a method. Too much composition API has ruined my judgement 😄. Fixed in f8d0728.